### DISCUSSION TITLE
migration doc not found

### DISCUSSION THREAD

**Discussion by FreTimmerman** [react-hook-form/react-hook-form#9184] [General] [answered]
see https://react-hook-form.com/migrate-v6-to-v7

it gives error 404: NOT_FOUND

**Comment by fahadsohail482** [comment, marked as answer]
@FreTimmerman docs for v5/v6 moved to 

[https://legacy.react-hook-form.com/migrate-v6-to-v7/](https://legacy.react-hook-form.com/migrate-v6-to-v7/) Please use this.

@bluebill1049 a suggestion is to add the migration guide link to the v7 docs as well.

**Reply by MetaMmodern** [reply]
A suggestion is to also optimize SEO to not lead whoever searches for this to old link.

**Reply by bluebill1049** [reply]
thanks gus, i am working on the redirect for it

**Reply by bluebill1049** [reply]
couldn't figure out a simple redirect with Gatsby... post a question there: https://github.com/gatsbyjs/gatsby/discussions/36790 if you guys know please send a pr :)

**Reply by MetaMmodern** [reply]
@bluebill1049 you can add a button with a link as a a temp solution (instead of showing 404). unless you figure out autoredirect

**Reply by bluebill1049** [reply]
yea, I have added a client-side redirect for now. i wish this is easier with gatsby
